eng/clipack/Common.projitems

```diff
@@ -0,0 +1,40 @@
+<Project>
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+    <_PublishOutputDirectory>$(ArtifactsBinDir)Aspire.Cli/$(Configuration)/$(TargetFramework)/$(CliRuntime)/publish/</_PublishOutputDirectory>
+  </PropertyGroup>
+
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
+
+  <Target Name="Build" />
+
+  <Target Name="BeforeBuild" BeforeTargets="Build">
+    <ItemGroup>
+      <AdditionalProperties Include="Configuration=$(Configuration)" />
+      <AdditionalProperties Include="TargetFramework=$(TargetFramework)" />
+      <AdditionalProperties Include="RuntimeIdentifier=$(CliRuntime)" />
+      <AdditionalProperties Include="PublishSingleFile=true" />
+      <AdditionalProperties Include="SelfContained=true" />
+      <!-- <AdditionalProperties Include="GenerateDocumenationFile=false" /> -->
+    </ItemGroup>
+
+    <MSBuild Projects="../../src/Aspire.Cli/Aspire.Cli.csproj" Targets="publish" Properties="@(AdditionalProperties)" />
+
+    <ItemGroup>
+      <_PublishItems Include="$(_PublishOutputDirectory)**/*" />
+    </ItemGroup>
+
+    <!-- Throw an error if _PublishItems is empty. -->
+    <Error Condition="'@(_PublishItems)' == ''"
+      Text="No files were found to pack. Ensure that the project $(MSBuildProjectFullPath) has a publish target defined." />
+
+    <MakeDir Directories="$(CliPublishedArtifactsOutputDir)/$(CliRuntime)" />
+    <ZipDirectory
```

Comment:
I know this is how we do it for the dashboard, but since this will be customer facing we should change it such that only Windows builds use .zip files, but Linux artifacts should generate .tar files instead. Most Linux distros come with utilities to extract tars, but not zips so we should avoid having this as a requirement.

Reply:
Hm I can't reference System.IO.Compression and System.Formats.Tar from the inline task to make this work. I'll create a new tasks project for this, and use that.